src/main/java/ua/lviv/iot/parkingServer/model/Parking.kt

```diff
@@ -1,15 +1,15 @@
 package ua.lviv.iot.parkingServer.model
 
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
 
-data class Parking(
-    override var id: Long,
-    var location: String,
-    var tradeNetwork: String,
-    var countOfParkingSpots: Int
-) : CsvData {
-
-    override fun getHeaders(): String =
-        listOf("Id", "Location", "Trade Network", "Count of parking spots").joinToString(separator = ", ")
 
-    override fun toCSV(): String = "$id, $location, $tradeNetwork, $countOfParkingSpots"
-}
+@Document("parking")
+data class Parking(
+    @Id
+    var id: String = ObjectId.get().toString(),
```

Comment:
you can omit declaring id
just do it lateinit , mongo will initialize it for you

Reply:
done